margin.feature

```diff
@@ -261,7 +261,7 @@ Feature: Margin Protocol
       | Alice | $200   | $1515       | 0_013201320132013201 | $-1315      | $-4800        |
     Then margin pool info are
       | ENP                  | ELL                  | Required Deposit |
-      | 0_976897689768976898 | 0_976897689768976898 | 0                |
+      | 1_409523809523809523 | 1_409523809523809523 | 0                |
```

Comment:
the new algorithm shouldn't change the calculated by this much?

Reply:
`equity= Fixed128(14800.000000000000000000), net_position =Fixed128(10500.000000000000000000)`
`net_position = pool.long.base_amount * curr_price_usd = 5000 * 2.2 = 10500`
so `equity/net_position = 1.4095238095238096`

Reply:
https://github.com/laminar-protocol/laminar-chain/pull/261/files#diff-e3eae8d2747e9b8a605686a84d86ba7dL903
The original calculation is a fixed value `leveraged_debits_in_usd` and will not change due to price changes